spray-aws/src/main/scala/com/sclasen/spray/aws/SprayAWSClient.scala

```diff
@@ -123,10 +136,22 @@ abstract class SprayAWSClient(props: SprayAWSClientProps) {
     request
   }
 
-  def response[T](response: HttpResponse)(implicit handler: HttpResponseHandler[AmazonWebServiceResponse[T]]): Either[AmazonServiceException, T] = {
+  def response[T](response: HttpResponse)(
+    implicit handler: HttpResponseHandler[AmazonWebServiceResponse[T]],
+             actorSystem: ActorSystem,
+             materializer: ActorMaterializer
+  ): Either[AmazonServiceException, T] = {
+    import scala.util.Try
+
     val req = new DefaultRequest[T](props.service)
     val awsResp = new AWSHttpResponse(req, null)
-    awsResp.setContent(new ByteArrayInputStream(response.entity.data.toByteArray))
+    val bs: Try[ByteString] = Try(Await.result(response.entity.toStrict(timeout.duration).map { _.data }, timeout.duration))
```

Comment:
@pepegar We need to get rid of this Await or it will be blocking. I guess this must be a semantic change between spray and akka-http in the handling of the HttpResponse. I guess we need to wrap this into a Future somehow (should be OK since all parts of the external API returns Futures).